src/main/java/github/scarsz/discordsrv/objects/managers/link/JdbcAccountLinkManager.java

```diff
@@ -53,7 +53,9 @@
 @SuppressWarnings("SqlResolve")
 public class JdbcAccountLinkManager extends AbstractAccountLinkManager {
 
-    private final static Pattern JDBC_PATTERN = Pattern.compile("([a-z]+)://(.+):(.+)/([A-z0-9]+)"); // https://regex101.com/r/7PSgv6
+    // more accurate RegEx for easier use later on and more accurate matches:
+    private final static String connectionRegExPattern = '^(?<DBProtocol>[a-zA-Z]+):(?<DBEngine>[a-zA-Z]+):\/\/(?<DBHost>.+):(?<DBPort>[0-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])\/(?<DBName>[a-zA-Z0-9_]+)(?<DBParameters>\?[a-zA-Z0-9]+=.[a-zA-Z0-9]+(?:&[a-zA-Z0-9]+=[a-zA-Z0-9]+)*)?$'
+    private final static Pattern JDBC_PATTERN = Pattern.compile(connectionRegExPattern);
```

Comment:
Using the wrong quotes, unnecessary escapes (for Java) and semicolon at the end, we can also reduce this to one variable since we only need the Pattern.

```suggestion
    private final static Pattern JDBC_PATTERN = Pattern.compile("^(?<DBProtocol>[a-zA-Z]+):(?<DBEngine>[a-zA-Z]+)://(?<DBHost>.+):(?<DBPort>[0-9]{1,4}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])/(?<DBName>[a-zA-Z0-9_]+)(?<DBParameters>?[a-zA-Z0-9]+=.[a-zA-Z0-9]+(?:&[a-zA-Z0-9]+=[a-zA-Z0-9]+)*)?$");
```